src/cmds/scripts/pbs_postinstall.in

```diff
@@ -463,7 +463,15 @@ elif [ $INSTALL_PACKAGE != client ] ; then
 		fi
 		cp ${PBS_EXEC}/libexec/pbs_init.d $initscript
 		rm -f /etc/rc.d/rc?.d/*pbs
-		if [ -x /sbin/chkconfig ] ; then 
+		if [[ `pidof systemd` ]]; then
```

Comment:
You use single brackets for this test in other scripts. I don't think double brackets are required here.